docs/reference/search/request/preference.asciidoc

```diff
@@ -65,3 +85,22 @@ GET /_search?preference=xyzabc123
 ------------------------------------------------
 // CONSOLE
 
+NOTE: The `_only_local` preference guarantees only to use shard copies on the
+local node, which is sometimes useful for troubleshooting. All other options do
+not _fully_ guarantee that any particular shard copies are used in a search,
```

Comment:
I think `_only_nodes` has the same so you won't need to route your request to the node in question